### DISCUSSION TITLE
CHANGELOG.md not getting pushed for protected branch

### DISCUSSION THREAD

**Discussion by Nirm01** [semantic-release/semantic-release#2995] [Configuration]
at async pluginsConfigAccumulator.<computed> [as prepare] (file:///usr/local/lib/node_modules/semantic-release/lib/plugins/index.js:87:11) {
  shortMessage: 'Command failed with exit code 1: git push --tags '',
  command: 'git push --tags '',
  exitCode: 1,
  signal: undefined,
  signalDescription: undefined,
  stdout: '',
  stderr: 'remote: GitLab: You are not allowed to push code to protected branches on this project.\n' +
  failed: true,
  timedOut: false,
  isCanceled: false,
  killed: false,
  pluginName: '@semantic-release/git'
}

**Comment by travi** [comment]
you need to use a token with appropriate permissions to push to your target branch. you can avoid this complexity if you choose not to make commits during your release process (we recommend against using the git and changelog plugins if you can instead leverage the release notes built into your git host)
